addons/website_livechat/static/src/web/livechat_channel_info_list_patch.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+    <t t-name="website_livechat.channelInfoList" t-inherit="im_livechat.channelInfoList" t-inherit-mode="extension">
```

Comment:
The template was badly named, it's been renamed to `im_livechat.LivechatChannelInfoList`.